### ISSUE TITLE
Tons of work being performed when setting status on entry

### ISSUE THREAD

**Issue by nitech** [spicywebau/craft-neo#240]
#### Description
When I select one or more entries in the entries list and select "Disabled for site", the web server performs a ton of work. When I review the web.log, I am not really able to access the web.log - only the web.log.1, 2 etc. because the web.log is still locked by the web server. 

![image](https://user-images.githubusercontent.com/1279436/62850858-94693900-bce4-11e9-8c7f-95305952646c.png)

I have attached the web.log.1-file so you can see what's going on. 
[web.log.1.zip](https://github.com/spicywebau/craft-neo/files/3491363/web.log.1.zip)

#### Other information
- Neo version: 2.4.2
- Craft version: 3.2.9 
- Is the affected Craft installation multi-site: Yes
- (if multi-site) Are the affected Neo fields set to manage blocks on a per-site basis: No

**Comment by nitech** [comment]
This also happens when I change propagation method (I changed to: "Save entries to all sites enabled for this section")

![Uploading image.png…]()

When I look at jobs, I can see one entry that is being increased +=1 on `attempt` every x minutes: 

```
O:34:"craft\queue\jobs\UpdateSearchIndex":6:{s:11:"elementType";s:20:"craft\elements\Entry";s:9:"elementId";s:3:"108";s:6:"siteId";s:1:"*";s:11:"description";N;s:30:" craft\queue\BaseJob _progress";i:0;s:35:" craft\queue\BaseJob _progressLabel";N;}
```

At the moment I have two attempts. If I look in `queue.log` I see two `[error]` logged, which corresponds with the number of retries on the job. 

I created an issue on oEmbed too. I am starting to believe this issue is related to oEmbed. 
https://github.com/wrav/oembed/issues/17
